sympy/matrices/dense.py

```diff
@@ -68,8 +68,16 @@ def __getitem__(self, key):
         >>> m[::2]
         [1, 3]
         """
-        if isinstance(key, tuple):
-            i, j = key
+        if isinstance(key, tuple) or key is Ellipsis:
+            if key is Ellipsis:
+                i = j = slice(None, None, None)
+            else:
+                i, j = key
+            if (i is Ellipsis) or (j is Ellipsis):
```

Comment:
Thanks for reviewing this PR. If only one among `i` or `j` is `Ellipsis`, then this line will have use.
#### Example
```
assert X[2, ...] == MatrixSlice(X, 2, (0, m))
```